src/main/java/roomescape/dto/TimeResponse.java

```diff
@@ -0,0 +1,18 @@
+package roomescape.dto;
+
+import java.time.format.DateTimeFormatter;
+import roomescape.domain.Time;
+
+public record TimeResponse(
+    Long id,
+    String time
+) {
+    private static final DateTimeFormatter TIME_FORMATTER = DateTimeFormatter.ofPattern("HH:mm");
+
+    public static TimeResponse from(Time time) {
+        return new TimeResponse(
+            time.getId(),
+            time.getTime().format(TIME_FORMATTER)
+        );
+    }
+}
```

Comment:
`@JsonFormat(pattern = "HH:mm")`을 사용하면 다음과 같이 코드를 변경할 수 있어요.
```suggestion
public record TimeResponse(
    Long id,
    @JsonFormat(pattern = "HH:mm")
    LocalTime time
) {

    public static TimeResponse from(Time time) {
        return new TimeResponse(
            time.getId(),
            time.getTime()
        );
    }
}
```

Reply:
LocalTime 타입을 그대로 유지하면서도 포맷만 간단하게 조정할 수 있어서 훨씬 간결하고 명확해졌어요!